apps/proposer/src/swagger.yaml

```diff
@@ -0,0 +1,373 @@
+openapi: 3.0.0
+info:
+  title: Proposer API
+  description: >
+    yey
+  version: 0.0.1
+servers:
+  - url: http://localhost:8092
+    description: Localhost
+tags:
+  - name: service
+    description: Information about the service
+  - name: transaction
+    description: Transaction routes
+  - name: proposer
+    description: Stuff about this proposer
+  - name: block
+    description: Actions on blocks
+paths:
+  /healthcheck:
+    get:
+      tags:
+        - service
+      summary: Healthcheck
+      description: I think it's already built-in in httputils
+      responses:
+        '200': # status code
+          description: Should return a 200 if alive and well
+
+  /transaction:
```

Comment:
Hey @signorecello, we could take advantage that we are refactoring things and move to the Restful pattern, the endpoints initial word are in plural.